tests/e2e/utils/activate-plugins.js

```diff
@@ -46,3 +48,27 @@ export async function activatePlugins( ...plugins ) {
 
 	await switchUserToTest();
 }
+
+/**
+ * Deactivates installed plugins.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Array.<string>} plugins Plugin slugs.
+ */
+export async function deactivatePlugins( ...plugins ) {
+	await switchUserToAdmin();
+	await visitAdminPage( 'plugins.php' );
+
+	for ( const plugin of plugins ) {
+		await page
+			.click( `tr[data-slug="${ plugin }"] input` )
+			.catch( () => {} );
+	}
+
+	await page.select( '#bulk-action-selector-top', 'deactivate-selected' );
```

Comment:
The second argument here is the only difference between these two functions. How about we extract a bulk plugin action function that these two use?

Reply:
Yep, good idea. Updated.